src/formpack/schema/fields.py

```diff
@@ -534,6 +533,21 @@ def get_labels(self, *args, **kwargs):
         return [self.name]
 
 
+class ValidationStatusCopyField(CopyField):
+
+    def format(self, val, lang=UNSPECIFIED_TRANSLATION, context=None):
+
+        if isinstance(val, dict):
+            if lang == XML_VALUES_AND_HEADERS:
```

Comment:
Let's use `if lang == UNSPECIFIED_TRANSLATION` per my other comment, even if this means changing KPI.

Reply:
done